docker-compose-files/openshift-compose/README.md

```diff
@@ -0,0 +1,48 @@
+### OpenShift import docker-compose
+
+
+Start ADB/CDK
+
+To get latest openshift edit Vagrantfile make `IMAGE_TAG` to `IMAGE_TAG="latest"`
+```bash
+vagrant up && vagrant provision
```

Comment:
Can this step be made if an existing Vagrant box with OSE 3.1 exists? Or it should be performed before the creation of the box?